docs/layouts/shortcodes/generated/pulsar_consumer_configuration.html

```diff
@@ -58,7 +58,7 @@
         </tr>
         <tr>
             <td><h5>pulsar.consumer.deadLetterPolicy.maxRedeliverCount</h5></td>
-            <td style="word-wrap: break-word;">0</td>
+            <td style="word-wrap: break-word;">(none)</td>
```

Comment:
Why do you change the default number here? It can be a breaking change for user-facing interface.

Reply:
This configure option is used on retry topic. The default option means no retry which is conflict with the `PULSAR_RETRY_ENABLE`.